lib/presentation/models/word_cloud/word_cloud_data.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/painting.dart';
+import 'dart:ui';
+
+class FlutterHashtag {
+  const FlutterHashtag(
+    this.hashtag,
+    this.color,
+    this.size,
+    this.rotated,
+  );
+  final String hashtag;
+  final Color color;
+  final int size;
+  final bool rotated;
+}
+
+class FlutterColors {
+  const FlutterColors._();
```

Comment:
Can be removed

Reply:
Addressed in commit 32d8fff0f0aab47e11fa37ebea8ad223f7ea29a9